theme/iframe.tpl.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * @file
+ * This is the template file for embedding Panopto iframe videos.
+ *
+ * Available variables: *EDIT*
+ * - $islandora_content: The rendered output of the viewer configured for
+ *   this module.
+ * - $islandora_dublin_core: The DC datastream object
+ * - $dc_array: The DC datastream object values as a sanitized array. This
+ *   includes label, value and class name.
+ * - $islandora_object_label: The sanitized object label.
+ * - $parent_collections: An array containing parent collection(s) info.
+ *   Includes collection object, label, url and rendered link.
+ *
+ */
+?>
+<iframe src="https://pitt.hosted.panopto.com/Panopto/Pages/Embed.aspx?id=<?php print $url?>
+&autoplay=<?php print $autoplay?>
```

Comment:
Do these newlines not get injected into the `src` URL?

Reply:
Nope. I have checked.